Spotkanie3_Zadania/String/String/Program.cs

```diff
@@ -0,0 +1,267 @@
+            for (int i = 0; i < 20; i++) sb.AppendLine(i.ToString());
+            Console.WriteLine("\nPrint below string use: 'StringBuilder for concatenation in tight loops'");
+            Console.WriteLine(sb.ToString());
+
+            //Concatenate string using String.Concate method
+            string first = "CokolwiekDoPrzykładu";
+            Console.WriteLine("\nPrint below string use: 'String.Concate method'");
+            string.Concat(first, word);
+        }
+
+        // compare word and usreName
+        static void CompareWord(string word1, string word2)
+        {
+            Boolean result1 = word1.Equals(word2);
+            Console.WriteLine($"Comparing whether \"{word1}\" is equal to \"{word2}\":\t {result1}");
+        }
+
+        //returns the position of the specified character in the string
+        static void ReturnPositionOfIndicatedCharacter(string word)
+        {
+            char letterLookingFor;
+            while (true)
+            {
+                Console.Write("We are looking for the index of the first occurrence of the letter you provided, dear user: ");
+                if (char.TryParse(Console.ReadLine(), out letterLookingFor))
+                {
+                    if (Char.IsNumber(letterLookingFor))
+                    {
+                        break;
+                    }
+                    else Console.WriteLine("Dear User, you have not entered a letter but a number, please try again.");
+                }
+                else Console.WriteLine("Dear User, you have not entered a letter, please try again.");
+            }
+            Console.Write($"We are looking for the index of the first occurrence of the letter '{letterLookingFor}': ");
+            if (word.IndexOf(letterLookingFor) < 0) Console.WriteLine($"The specified letter \"{letterLookingFor}\" is missing in the string: {word}");
+            else Console.WriteLine(word.IndexOf(letterLookingFor));
+        }
+
+        //returns substring of a string
+        static void ReturnsSubstringOfString(string word)
+        {
+            Console.Write("We print now substring which start from index what you want position: ");
+            int startIndex = EnterNumber();
+
+            Console.Write("We print now substring which end in what you want position: ");
+            int endIndex = EnterNumber();
+
+            Console.WriteLine($"We print now substring which start from index {startIndex} and end in {endIndex} position: {word.Substring(startIndex, endIndex)} ");
+        }
+
+        static int EnterNumber()
+        {
+            int numberCandidate;
+            while (true)
+            {
+                if (int.TryParse(Console.ReadLine(), out numberCandidate)) break;
+                else Console.Write($"Dear User, this is not a number, please try again: ");
+            }
+            return numberCandidate;
+        }
+
+
+        //splits the string into substring
```

Comment:
komentarz do usuniecia.
Funkcja nie jest uzyta

Reply:
Tyle nawaliłem tych funkcji że jedna mi się zapodziała, a tak właściwie czemu komentarz do usunięcia?

Reply:
Komentarze w C# nie sa dobra praktyka, jesli cos wymaga komentarza, to jest to sygnal, ze trzeba ulepszyc nazwe, albo uproscic logike, albo zrobic jakis refaktor.
Tu miales komentarz `//splits the string into substring` a zaraz funkcje z podobna nazwa